### PR TITLE
PWGCF: flow-pbpb: apply NUA and NUE weights

### PR DESCRIPTION
**Pull request by Luzhiyongg**
small changes: 

1. change ccdb url; 
2. using enum for index;

major changes: 

1. change pt bins; 
2. add code to produce NUA
3. add code to read NUA and NUE

### CODE REVIEW

**Comment by Luzhiyongg** [comment]
@victor-gonzalez Hi victor, sorry to disturb. Could you take a look at the code and approve the merging process?

File: PWGCF/Flow/Tasks/FlowPbPbTask.cxx [hunk 1]
```diff
@@ -283,8 +307,48 @@ struct FlowPbPbTask {
...
+      if (mEfficiency == nullptr) {
+        LOGF(fatal, "Could not load efficiency histogram for trigger particles from %s", cfgEfficiency.value.c_str());
+      }
+      LOGF(info, "Loaded efficiency histogram from %s (%p)", cfgEfficiency.value.c_str(), (void*)mEfficiency);
+    }
+    correctionsLoaded = true;
+  }
+
+  bool setCurrentParticleWeights(float& weight_nue, float& weight_nua, const float& phi, const float& eta, const float& pt, const float& vtxz)
+  {
```
Comment (victor-gonzalez) [inline PWGCF/Flow/Tasks/FlowPbPbTask.cxx:332]: Please, for the next iteration, don't pass `phi`, `eta`, `pt` nor `vtxz` by reference pass them by value, i.e. `..., float phi, ...`

File: PWGCF/Flow/Tasks/FlowPbPbTask.cxx [hunk 2]
```diff
@@ -305,11 +372,16 @@ struct FlowPbPbTask {
     for (auto& track : tracks) {
       double pt = track.pt();
       double eta = track.eta();
+      double phi = track.phi();
```
Comment (victor-gonzalez) [inline PWGCF/Flow/Tasks/FlowPbPbTask.cxx:375]: For the next iteration please, don't store the values unless you need them and don't convert them to `double`, they are stored as `float` in the tables
Don't be worried by repeating, for instance `track.pt()` several times. The compiler will take care of feasible optimizations

**Review by victor-gonzalez — APPROVED** [review]
Sorry I overlooked your PR
Please, for the next iteration have a look at my comments

**Pull request merged**